letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -351,16 +366,21 @@ def _find_best_vhost(self, target_name):
         :returns: VHost or None
 
         """
-        # Points 4 - Servername SSL
-        # Points 3 - Address name with SSL
-        # Points 2 - Servername no SSL
+        # Points 6 - Servername SSL
```

Comment:
I'm not sure it really matters, but Apache gives no preference to exact `ServerName/ServerAlias` vs those with wildcards ([source](https://httpd.apache.org/docs/current/mod/core.html#serveralias)). For this reason, I'm not sure we should either and perhaps they should have the same priority.